### PR TITLE
Update reverse SSH guide

### PR DESCRIPTION
**Pull request by dennisreimann**
A few additions and updates to make it work for me.

### CODE REVIEW

File: docs/ReverseSSHtunnel.md [hunk 1]
```diff
@@ -52,7 +66,7 @@ sudo nano /etc/ssh/sshd_config
 Make sure these entries are active (meaning there is no `#` at the beggining of the line).
 Alternatively, you can just paste these on the end of the file:
 
-```
+```text
```
Comment (Zaxounette) [inline docs/ReverseSSHtunnel.md:69]: text ?

**Review by Zaxounette — COMMENTED** [review]
Typos, wording and such ACK.

Just the following that I don't understand.

File: docs/ReverseSSHtunnel.md (hunk 1 above)
Comment (dennisreimann) [reply docs/ReverseSSHtunnel.md:69]: There's no other highlighting format for it, but we can go ahead and remove it – the result will be the same.

Comment (Zaxounette) [reply docs/ReverseSSHtunnel.md:69]: As you wish.
Just didn't understand the purpose, that's all.

Comment (dennisreimann) [reply docs/ReverseSSHtunnel.md]: Removed it :)

**Pull request merged**